packages/rocketchat-file-upload/server/lib/FileUpload.js

```diff
@@ -206,14 +206,17 @@ Object.assign(FileUpload, {
 		}
 
 		let { rc_uid, rc_token } = query;
+		const { rc_room_type } = query;
```

Comment:
this will need to change if we change to cookies

Reply:
Fixed.